site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -231,3 +232,7 @@ Calling other endpoints would result in a `401` response code.
 == Client Side API Keys
 
 include::docs/v1/tech/shared/_client-side-api-keys.adoc[]
+
+== Troubleshooting
```

Comment:
Should we just have this in one spot so we don't have duplicate doc?

Reply:
I thought it was useful to have 'troubleshooting for authentication' in the authentication section and then include this in the overview troubleshooting section too. Usually this is a good case for the include file.